scanner/src/lib/dataproducts/Battery_HealthData_v0.1.svelte

```diff
@@ -14,6 +14,7 @@
   import Divider from "$components/Common/Divider/index.svelte"
   import Title from "$components/Common/Title/index.svelte"
   import SubTitle from "$components/Common/SubTitle/index.svelte"
```

Comment:
This name is incorrect, and will crash on a case-sensitive filesystem.

![image](https://github.com/ioxiocom/ioxio-tags/assets/1296558/e0823ebd-d2ce-41e3-87a7-30f34f34dc95)

Reply:
![image](https://github.com/ioxiocom/ioxio-tags/assets/1296558/0fbf2cd1-c206-47d7-8074-d7001b75817b)